examples/web/vite-chat-app/src/App.tsx

```diff
@@ -37,7 +37,9 @@ export default function Home() {
             className={"block mb-2 text-sm font-medium text-gray-900 text-center"}
           >
             This app was configured to allow only authenticated users. <br />
-            Please select an existing Cognito user to continue.
+            This dropdown menu would be a sign-in portal in non-demo applications. 
```

Comment:
```suggestion
            In a real-world application, this would be a more typical login or SSO page. For demo purposes you can select one of our pre-configured users.
```